CPS-0023/README.md

```diff
@@ -0,0 +1,135 @@
+
+- **Token Donations and Deposits**: Anyone could donate or return funds in any CNT. Community members or partner organizations might send their native tokens (including stablecoins) to the treasury as contributions. This expands the treasury’s resource pool beyond ada – e.g. a DeFi project that raised funds in a stablecoin could return unused funds directly in that stablecoin to the treasury, rather than converting back to ada.
+
+- **Cross-Chain Collaboration**: A multi-asset treasury can strengthen partnerships with Cardano’s partner chains. These networks could contribute their native tokens to Cardano’s treasury as part of collaboration agreements. This creates financial bridges; for instance, if an IOU token from a sidechain or a governance token from a partner blockchain, which are represented as CNTs on Cardano, is contributed to Cardano’s treasury, it effectively links the ecosystems.
+
+Notably, multi-asset treasuries are not without precedent. Other blockchain ecosystems with on-chain governance have already implemented similar features. Polkadot, for example, has a treasury that holds DOT (its native coin) plus assets like USDC, USDT, and even tokens from its parachains. Cardano, coming later to on-chain governance, can leverage these lessons as a second mover – implementing a multi-asset treasury in a careful, informed way.
+
+## Goals
+
+The good solution to this CPS consists of 3 components:
+
+1. Well articulated proposals considering below:
+
+   - Amend the Cardano Constitution:
+
+     - To address potential other CNTs in the constitutions
+     - Any guardrails in practice for inclusion of CNTs
+     - Do we define net change limits in CNTs also
+
+   - A new CIP for multi asset treasury covering below areas:
+
+     - Exact specification to describe the ledger change
+     - Analysis to take network security into account
+     - Do we need new protocol parameters for CMAT
+     - What we use for threshold for listing & delisting if it introduces new governance actions
+     - Other ledger rule's technical consideration such as min ADA
+
+   - Update on existing CIPs such as [CIP-1694](https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694).
+
+2. Submit an Info Action of the CIP to gauge community sentiment if this is a requested change.
+
+3. Implementation of the CIP (likely including passing governance action to initial hardfork in updating ledger)
+
+### Out of Scope
+
+- The whitelisted assets to be included in the Cardano Multi Asset Treasury
+- The conversion and management of the multi-assets in the treasury
+
+## Open Questions
+
+### Spam Prevention - Only Allow Whitelisted Tokens
+
+Spam prevention as if only whitelisted tokens can send into the treasury. Do we want to build in this feature into CMAT?
+
+- For: Discussion in [(CIP159)](https://github.com/cardano-foundation/CIPs/pull/1061) thread acknowledges the need for spam prevention
```

Comment:
```suggestion
- For: Discussion in [(CIP-0159)](https://github.com/cardano-foundation/CIPs/pull/1061) thread acknowledges the need for spam prevention
```
Please follow the example everywhere else on the repository of either using the tag "CIP-0159" or the name "CIP 159".  Mixing these syntaxes makes text much hard to search.